jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6256,6 +6262,250 @@
 						"description": ""
 					},
 					"response": []
+				}
+				
+			]
+		},
+		{
+			"name": "s)  upgrade JDK test operations",
+			"description": "",
+			"item": [
+{
+					"name": "create new group - for JDK upgrade testing ",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);",
+									"postman.setEnvironmentVariable(\"JDKUpgradeGroupId\", jsonData.applicationResponseContent.id.id);",
+									"tests[\"Assert response body\"] = jsonData.applicationResponseContent.name === postman.getEnvironmentVariable(\"JDKUpgradeGroup\");",
+									"tests[\"Successful POST request\"] = responseCode.code === 201;",
+									"",
+									""
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/groups",
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{{JDKUpgradeGroup}}"
+						},
+						"description": "create group for JDKUpgradeGroup . This group will be used for JDK upgrade testing."
+					},
+					"response": []
+				},
+				{
+					"name": "create jvm for JDK Upgrade",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 201\"] = responseCode.code === 201;",
+									"var jsonData = JSON.parse(responseBody);",
+									"postman.setEnvironmentVariable(\"JDKUpgradeJvmId\", jsonData.applicationResponseContent.id.id);",
+									"tests[\"Assert response body\"] = jsonData.applicationResponseContent.jvmName === postman.getEnvironmentVariable(\"JDKUpgradeJvmName\");"
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/jvms",
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"jvmName\":\"{{JDKUpgradeJvmName}}\",\"groupIds\":[{\"id\":10,\"groupId\":{{JDKUpgradeGroupId}}}],\"hostName\":\"{{host}}\",\"statusPath\":\"/tomcat-power.gif\",\"systemProperties\":\"\",\"httpPort\":\"{{jdkUpgrade-HTTP-port}}\",\"httpsPort\":\"{{jdkUpgrade-HTTPS-port}}\",\"redirectPort\":\"{{jdkUpgrade-redirect-port}}\",\"shutdownPort\":\"{{jdkUpgrade-shutdown-port}}\",\"ajpPort\":\"{{jdkUpgrade-ajp-port}}\",\"userName\":\"\",\"encryptedPassword\":\"\",\"jdkMediaId\":{{JdkMediaId}},\"tomcatMediaId\":{{TomcatMediaId}}}"
```

Comment:
Please follow the camel case formatting for variable names.
Don't use hyphens in the variable names for the ports.